glib/src/subclass/interface.rs

```diff
@@ -3,25 +3,28 @@
 use std::{marker, mem};
 
 use super::{InitializingType, Signal};
-use crate::{prelude::*, translate::*, Object, ParamSpec, Type};
+use crate::{
+    gobject::traits::DynamicObjectRegisterExt, prelude::*, translate::*, Object, ParamSpec, Type,
```

Comment:
```suggestion
    prelude::*, translate::*, Object, ParamSpec, Type,
```
glib's prelude should include gobject's one